tests/test-project/src/test/editor/extensionEditor.test.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import {
+	ActivityBar,
+	EditorView,
+	ExtensionEditorView,
+	ExtensionEditorDetailsSection,
+	ExtensionsViewItem,
+	ExtensionsViewSection,
+	SideBarView,
+	ViewControl,
+	VSBrowser,
+	WebDriver,
+} from 'vscode-extension-tester';
+import * as pjson from '../../../package.json';
+
+describe('Extension Editor', function () {
+	let driver: WebDriver;
+	let viewControl: ViewControl;
+	let extensionsView: SideBarView;
+	let item: ExtensionsViewItem;
+
+	let section: ExtensionsViewSection;
+
+	let extensionEditor: ExtensionEditorView;
+	let extensionEditorDetails: ExtensionEditorDetailsSection;
+
+	before(async function () {
+		driver = VSBrowser.instance.driver;
+		viewControl = (await new ActivityBar().getViewControl('Extensions')) as ViewControl;
+		extensionsView = await viewControl.openView();
+		await driver.wait(async function () {
+			return (await extensionsView.getContent().getSections()).length > 0;
+		});
+
+		const view = await ((await new ActivityBar().getViewControl('Extensions')) as ViewControl).openView();
+
+		await view.getDriver().wait(async function () {
```

Comment:
I would recommend to decide between `driver = VSBrowser.instance.driver;` or `something.getDriver()..` and use selected across whole class